demoapps/simplest/proguard-rules.pro

```diff
@@ -0,0 +1,23 @@
+# Add project specific ProGuard rules here.
```

Comment:
Looks like this file is completed commented out. Could we remove it?

Reply:
Sure

Reply:
Done